### PR TITLE
added dexterity search extender behavior

### PR DESCRIPTION
**Pull request by reinhardt**
provides showinsearch and searchwords for dexterity types, see #57

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -159,7 +159,9 @@ Exclude from search and elevation
 ---------------------------------
 
 By default this add-on introduces two new fields to the default content types
-or any custom type derived from ATContentTypes.
+or any custom type derived from ATContentTypes. For dexterity content types,
+including plone.app.contenttypes, a behavior is provided which must be manually
+activated on the desired content types.
```
Comment (tisto) [inline README.rst]: @reinhardt shouldn't that be "on the desired content object"? Activation happens on the content object, not the type, right?

**Comment by tisto** [comment]
@reinhardt pull request looks good. Though, we should discuss how we could test this feature. I guess we could create a basic behavior test (that makes sure the behavior can be activated and used). We could also think about writing a high-level robot framework test that activates the behavior and makes sure the actual elevation works. This test could also be the documentation for that feature. With the new solr test fixtures it should be easy to set up a basic robot framework test setup. I'd be willing to help with the setup...

**Comment by tisto** [comment]
@reinhardt oh, and please rebase.

File: README.rst (hunk 1 above)
Comment (reinhardt) [inline README.rst]: It's a regular behavior, so I'm pretty sure you can do both. My understanding was that the usual way to activate a behavior is on a content type.

**Comment by tisto** [comment]
@reinhardt any news on this? I would like to merge but we need to rebase and write at least a basic test.

**Comment by reinhardt** [comment]
Sorry, haven't gotten around to it yet. This week seems unlikely as well. Will try to sneak it in next week.

**Comment by tisto** [comment]
@reinhardt any news on this?

**Comment by reinhardt** [comment]
Sorry again - not yet. Thanks for the reminder.

**Comment by reinhardt** [comment]
I've rebased now and added a first test. It's probably a little _too_ basic - could you have a look?

**Comment by reinhardt** [comment]
I'm working on a second test but it's currently failing and I don't see why. I've looked at 

https://github.com/simplesconsultoria/collective.behavior.contactinfo/blob/master/collective/behavior/contactinfo/docs/address.txt
and
https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/tests/test_behaviors_leadimage.py

for inspiration but tried to avoid the test browser. [Line 33](https://github.com/collective/collective.solr/commit/2d94ec5d4ec957ff3a50529a9409d8b5dccfc36f#diff-23abcea12dad8bd49ffb364bfcc1bafdR33) currently throws an "AttributeError: showinsearch". Any hints appreciated.

**Comment by tisto** [comment]
@reinhardt what's the status here? Travis is green. Is the test still failing? I would like to make a release in the next days and it would be nice to include your pull request.

**Comment by reinhardt** [comment]
It's still failing (it's skipped by default). I'd say the code itself is ready for use, but maybe it would be good to have another working test...
However I'll be pretty short on time for the foreseeable future I'm afraid.

**Comment by tisto** [comment]
@reinhardt ok, the please just rebase and I will have a look and merge...

**Comment by tisto** [comment]
Will close this pr for now. @reinhardt feel free to re-open at any time if you feel like it.

**Pull request closed without merging**